ocean_lib/test/test_config_provider.py

```diff
@@ -22,18 +22,18 @@ def test_set_config():
     assert ConfigProvider.get_config() == "foo config"
 
 
-def test_metadataCacheUri_version():
-    """Tests that the Aquarius URL can use the custom metadataCacheUri key."""
-    config_dict = {"metadataCacheUri": "http://ItWorked.com", "network": "ganache"}
+def test_metadataStoreUri_version():
+    """Tests that the Aquarius URL can use the custom metadataStoreUri key."""
```

Comment:
please adjust these docstrings to reflect the new purpose of these tests.

Reply:
Done.